vr/common/templates/proc.conf

```diff
@@ -1,7 +1,8 @@
 [program:%(container_name)s]
-command=%(runner)s run %(proc_yaml_path)s
+command=bash -c "if [[ -f %(proc_yaml_path)s ]] || [[ -f %(container_path)s ]] ; then %(runner)s run %(proc_yaml_path)s; fi"
 autostart=true
 autorestart=true
+startretries=3
```

Comment:
According to the docs startretries default value is 3. Is this in here because the default value is different in the version that we are using internally?